### ISSUE TITLE
Import open3d and opencv-python libraries

### ISSUE THREAD

**Issue by polzaco** [tmontaigu/CloudCompare-PythonRuntime#95]
Hey! I have an issue installing open3d and opencv-python libraries. I have tried installing those from "Manage packages with pip" and I have Errors with the installation. Also I have tried installing by cmd:
"D:\CloudCompare\plugins\Python\python.exe" -m pip install opencv-python
"C:\Program Files\CloudCompare\plugins\Python\python.exe" -m pip install open3d
With this is possible install them, but then it doesnt recognize the modules idk why.
ModuleNotFoundError: No module named 'cv2'

**Comment by tmontaigu** [comment]
#44

**Comment by tmontaigu** [comment]
Normaly, installing with a cmd or cloudcompare launched with admin rights should fix this

**Comment by polzaco** [comment]
Thank you tmontaigu. I have tried that you said but it doesnt still work, but I have seen in the picture appended that there is 
~umpy.libs folder in the directory c:\program files\cloudcompare\plugins\python\lib\site-packages. Also there were partitions like ~harset_normalizer, ~illow and ~ydantic, that means there are partitions of the packages numpy, charset_normalizer, pillow and pydantic when you try to install some libraries.

![Captura de pantalla (1)](https://github.com/tmontaigu/CloudCompare-PythonRuntime/assets/107433987/be5dbeb4-17b8-40c8-b9be-6706acde517d)

I have deleted this folders from the directory (c:\program files\cloudcompare\plugins\python\lib\site-packages) and uninstalled from "Manage packages with pip" and then install open3d and opencv-python with admin rights and now works.
